devtools/gradle/src/functionalTest/resources/add-extension-multi-module-kotlin-dsl/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -0,0 +1,5 @@
+distributionBase=GRADLE_USER_HOME
+distributionPath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.1-bin.zip
```

Comment:
Do we need specific gradle versions here? Let's use whatever the gradle plugin project is using instead.

Reply:
I will remove gradle wrapper related files from test resources directory. They were created when I tested this project manually.